backend/src/index.ts

```diff
@@ -48,6 +48,8 @@ type ChatSocket = Socket & {
   sessionId: string;
 };
 
+var reactions = {};
```

Comment:
`var` -> `const`

Reply:
na pewno const? reactions to taki lokalny stan który jest często modyfikowany

Reply:
ale generalnie mi też to var się nie podoba - w jaki sposób można "ładniej" reprezentować lokalny stan w node js?

Reply:
linter sugeruje wsadzenie tam consta - zweryfikowałem to i działa (jeżeli jednak nie przyjmiemy tutaj konwencji wprowadzenia stałej to proponuję zmianę na let)

Reply:
oki dałem consta

Reply:
`const` i obiekt to porównałbym do `private final` i `ArrayList` w javie. Referencja jest niemutowalna, natomiast obiekt na który referencja wskazuje już jest mutowalny